app/view/show.py

```diff
@@ -91,8 +117,10 @@ def show_group(avg):
         'cves': cves,
         'advisories': advisories
     }
+
     return render_template('group.html',
                            title='{}'.format(group.name),
+                           bug_data=add_bug_link(cves, pkgs, group.severity),
```

Comment:
we should pass the group itself to the get_bug_data function so we can populate the render_template of it with group specific data. This way we could (f.e.) add a back reference to the AVG view of the tracker (or maybe use other data from the group itself)